components/Map/Map.tsx

```diff
@@ -51,18 +107,56 @@ const defaultViewPort: ViewState = {
   pitch: 0,
 }
 
+const legendLine = category => (
+  <div>
+    <div className="circle"> </div> {category.description}
+    <style jsx>{`
+      .circle {
+        background: hsl(${category.hue}, 100%, 50%);
+        width: 10px;
+        height: 10px;
+        border-radius: 50%;
+        float: left;
+      }
+    `}</style>
+  </div>
+)
+
+const legend = () => (
```

Comment:
This can move to its own file as a component, other than this amazing work 🙌

Reply:
might pick this up later!